examples/pytorch/question-answering/README.md

```diff
@@ -57,6 +57,28 @@ f1 = 88.52
 exact_match = 81.22
 ```
 
+### Fine-tuning T5 on SQuAD2.0
+
+This example code fine-tunes T5 on the SQuAD2.0 dataset.
+
+```bash
+python run_seq2seq_qa.py \
+  --model_name_or_path t5-small \
+  --dataset_name squad_v2 \
+  --context_column context \
+  --question_column question \
+  --answer_column answer \
+  --do_train \
+  --do_eval \
+  --per_device_train_batch_size 12 \
+  --learning_rate 3e-5 \
+  --num_train_epochs 2 \
+  --max_seq_length 384 \
+  --doc_stride 128 \
+  --output_dir /tmp/debug_seq2seq_squad/
+```
+
+
```

Comment:
Similar to the BERT example, you could provide the corresponding results here, something like:

```suggestion
Training with the previously defined hyper-parameters yields the following results:

f1 = 88.52
exact_match = 81.22
```